indico/modules/events/papers/templates/display/call_for_papers.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% from 'events/papers/display/_common.html' import render_cfp_infoline %}
+{% from 'events/papers/display/_paper_list.html' import render_user_papers_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {%- trans %}Call for Papers{% endtrans -%}
+{% endblock %}
+
+{% macro render_timestamp(label, date, tz, completed=false, alternative_text=none) %}
+    <li class="{{ 'completed' if completed }}">
+        <div class="label">{{ label }}</div>
+        <div class="timestamp flexcol f-a-center">
+            {% if date %}
+                <time datetime="{{ date.isoformat() }}">
+                    {{- date|format_datetime(timezone=tz) -}}
+                </time>
+            {% elif alternative_text %}
+                <span>{{ alternative_text }}</span>
+            {% endif %}
+        </div>
+    </li>
+{% endmacro %}
+
+{% macro render_timeline_section(cfp) %}
+    {% set tz = cfp.event.display_tzinfo %}
+    <section>
+        <ul class="cfa-timeline flexrow f-a-center f-j-center">
```

Comment:
what I'm saying is that this should probably be renamed and moved (currently in _cfa_page.scss) when it's used here. cc @OmeGak